### PR TITLE
[ci skip] Add maintainers

### PR DESCRIPTION
**Pull request by consideRatio**
To help with maintenance chores of dask-gateway and dask-gateway-server defined in https://github.com/dask/dask-gateway, I'm opening this PR.

I encourage others in the Dask team willing to contribute with conda-forge expertice to add themselves as maintainers as well by adding a code suggestion to this PR.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -152,3 +152,4 @@ about:
 extra:
   recipe-maintainers:
     - jcrist
+    - consideratio
```
Comment (consideRatio) [inline recipe/meta.yaml]: @jacobtomlinson @TomAugspurger @martindurant for example could perhaps be willing to help with conda-forge maintenance chores?

Comment (martindurant) [reply recipe/meta.yaml]: Happy to be included

Comment (consideRatio) [inline recipe/meta.yaml]:
```suggestion
    - consideratio
    - martindurant
```

Comment (TomAugspurger) [inline recipe/meta.yaml]:
```suggestion
    - consideratio
    - martindurant
    - TomAugspurger
```

Comment (jacobtomlinson) [reply recipe/meta.yaml]:
```suggestion
    - consideratio
    - martindurant
    - TomAugspurger
    - jacobtomlinson
  ```

Comment (martindurant) [reply recipe/meta.yaml]: I believe only @jcrist can accept these suggestions and merge the PR

**Comment by jcrist** [comment]
@conda-forge-admin, please rerender

**Comment by jcrist** [comment]
Hmmm, perhaps the conda bot is down since github is degraded? I'm just trying to see if the build failures here are fixed - there's an unsatisfiable requirements error on some of the builds.

**Comment by consideRatio** [comment]
> Hmmm, perhaps the conda bot is down since github is degraded? I'm just trying to see if the build failures here are fixed - there's an unsatisfiable requirements error on some of the builds.

I've not looked into the details yet, but I suspect that there are issues for py36 and py37 since the dask ecosystem has moved away from supporting them. We should update the recipe to do py38-py10.

I can look into this, but for now, agree to merge and grant us to be maintainers of this feedstock @jcrist ?

**Comment by jcrist** [comment]
> I can look into this, but for now, agree to merge and grant us to be maintainers of this feedstock

I'm happy to merge and add you as maintainers, I'm just worried it'll kick off a new build of the package. If it won't, then this seems fine to me.

**Comment by consideRatio** [comment]
Ah thanks for the clarification @jcrist! I'll need to learn a bit more about conda-feedstocks myself to sort this out, I'll ping back at the time I know enough to suggest what seem to make sense!

**Comment by martindurant** [comment]
You can always ping the conda-forge admins for information when needed, they are responsive and helpful.

**Comment by martindurant** [comment]
The release on pypi normally triggers an attempt by conda-forge to rebuid for the new version - usually within a couple of hours.  That build should also have any updates to the feedstock from conda-forge itself. We can add maintainers in that same PR, when it arrives.

**Comment by consideRatio** [comment]
I've read up a bit on the documentation. I conclude from reading [this docs on adding a maintainer](https://conda-forge.org/docs/maintainer/updating_pkgs.html#updating-the-maintainer-list) and testing it out in #24 --> #25, I think it should be safe to merge this PR as it now is as we now have `[ci skip]` to the title of this PR and in the commit like the bot did in #25.

**Pull request closed without merging**